### PR TITLE
feat: enabled manual height adjustment for bounty description box

### PR DESCRIPTION
**Pull request by okhot**
### Describe your changes
enabled manual height adjustment for bounty description box

### Preview
https://www.loom.com/share/9a26a03ecaf94ad0bdc2d1d222933f4f?sid=803ca758-716e-4ca4-a4e4-fc498fae871d

### Type of change
Feature

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)

### Checklist before requesting a review
- [X] I have performed a self-review of my code
- [X] I have tested on Chrome and Firefox
- [X] I have provided a screenshot or recording of changes in my PR if there were updates to the frontend

### CODE REVIEW

**Comment by ecurrencyhodler** [comment]
Great work. Screenshot looks good. All checks passed. Let's do a code review.

File: frontend/app/src/components/form/bounty/index.tsx [hunk 1]
```diff
@@ -444,8 +444,9 @@ function Form(props: FormProps) {
                           .map((item: FormField) => (
                             <Input
                               {...item}
+                              type={item.name === 'description' ? 'textarea' : item.type}
```
Comment (kevkevinpal) [inline frontend/app/src/components/form/bounty/index.tsx:447]: instead of cheking `item.name === 'description'` twice lets define it as a boolean above and use it in the two places

File: frontend/app/src/components/form/inputs/TextAreaInput.tsx [hunk 2]
```diff
@@ -97,7 +97,7 @@ export default function TextAreaInput({
         }}
         className={active ? 'euiFormRow_active' : (value ?? '') === '' ? '' : 'euiFormRow_filed'}
         border={borderType}
-        label={labeltext}
+        label={labeltext === 'Description' ? `${labeltext} *` : labeltext}
```
Comment (kevkevinpal) [inline frontend/app/src/components/form/inputs/TextAreaInput.tsx:100]: we should try to keep logic that is not related to the common component out of here

**Comment by okhot** [comment]
@kevkevinpal alright!

**Comment by ecurrencyhodler** [comment]
All checks pass. Great job. @elraphty let's get a code review.

**Comment by elraphty** [comment]
@okhot I can't seem to increase the description height on my chrome browser on Mac [Loom Video](https://www.loom.com/share/d6dd123fd4f94ba1819c0d961d68d097)

**Comment by elraphty** [comment]
@okhot it works on firefox, but doesn't work on chrome [Loom Video](https://www.loom.com/share/b4ffcc0230ee443fa7ee4734c33f7c20), make it work on chrome.

**Comment by okhot** [comment]
@elraphty Here's a demo on Chrome https://www.loom.com/share/832a381e81c1408d9c80c99b046761af?sid=6140fa11-93aa-44c5-9d6e-bff20092d486

**Comment by elraphty** [comment]
> @elraphty Here's a demo on Chrome https://www.loom.com/share/832a381e81c1408d9c80c99b046761af?sid=6140fa11-93aa-44c5-9d6e-bff20092d486

It doesn't work on Mac's chrome, I think you have to make some research regarding this

**Comment by okhot** [comment]
@elraphty Here's a demo on Mac
https://www.loom.com/share/17cde2490a9441ac85c90bdd4c488f33?sid=92bfe385-bb17-4018-b34e-41125778f133

**Comment by elraphty** [comment]
> @elraphty Here's a demo on Mac https://www.loom.com/share/17cde2490a9441ac85c90bdd4c488f33?sid=92bfe385-bb17-4018-b34e-41125778f133

Nice Job @okhot, I tested it on my Chrome Mac and it works now. I'm merging right away.

**Comment by ecurrencyhodler** [comment]
tested and it works. GJ! Paid!
